### ISSUE TITLE
Word Tokenization Benchmark Removal

### ISSUE THREAD

**Issue by p16i** [PyThaiNLP/pythainlp#428]
I think about having the benchmark in PyThaiNLP again, and I think it might be better to remove it from the repository, at least the procedure should be peer-reviewed.

Sorry again for your inconvenience! I should have thought about it.

**Comment by bact** [comment]
- It would be nice to have a handy standard benchmarking tool and I think PyThaiNLP as a project can be a good place to host and support it.
- But to have it as part of standard pythainlp module or not is another question.
- I think the current approach is fair enough, by have it as an optional install. Not forcing everybody to have it, but if anyone wants it, it can be installed easily.
- Another way is to have it as a separated package that people can, say, just install it by `pip install wordsegbench` or something similar

**Comment by bact** [comment]
btw, after clean the CLI `thainlp data` and corpus download stuffs, I plan to thinking about how to make the benchmark available via `thainlp` command line.

**Comment by p16i** [comment]
@cstorm125  suggested that we have the script there but with a disclaimer. I actually like the idea. What do you think?

**Comment by cstorm125** [comment]
Yes I agree with disclaimer. Then once @mrpeerat is ready, he can publish the updated benchmarks.

**Comment by p16i** [comment]
I guess we can close this issue for now.
